Packs/Jira/Integrations/JiraV2/JiraV2.py

```diff
@@ -366,6 +367,9 @@ def get_issue_fields(issue_creating=False, **issue_args):
     if issue_args.get('labels'):
         issue['fields']['labels'] = issue_args['labels'].split(",")
 
+    if issue_args.get('components'):
+        issue['fields']['components'] = issue_args['components'].split(",")
```

Comment:
From the examples in the API it seems that components are complex objects.
like this ?`{"set" : [{"name" : "Engine"}, {"name" : "Trans/A"}`
![image](https://user-images.githubusercontent.com/7270217/88489495-30935b00-cf9d-11ea-89af-cc8aa59d9bde.png)

Reply:
also according to the Jira docs seems the only example I see for updating components is to have it under `update`  and not under `fields` . Or you checked and it just works?
https://developer.atlassian.com/server/jira/platform/jira-rest-api-examples/#editing-an-issue-examples